### DISCUSSION TITLE
add integration with collabora

### DISCUSSION THREAD

**Discussion by ghost** [filebrowser/filebrowser#1651] [Ideas]
could you add collabora integration with filebrowser to edit office document?

here collabora shows how to make an integration with any web, collabora needs an iframe on filebrowser  to edit file.

https://sdk.collaboraonline.com/docs/How_to_integrate.html

**Comment by ghost** [comment]
an example with simple web page:

https://github.com/CollaboraOnline/collabora-online-sdk-examples/tree/master/webapp/php

**Comment by nook24** [comment]
This kind of relates to https://github.com/filebrowser/filebrowser/discussions/1272 so i thought it would be a good idea to add a link.
